configure.in

```diff
@@ -41,6 +41,10 @@ AC_ARG_WITH(jack,
             AS_HELP_STRING([--with-jack], [Enable support for JACK @<:@autodetect@:>@]),
             [with_jack=$withval])
 
+AC_ARG_WITH(pulseaudio,
```

Comment:
We will need to run autoconf to generate the new "configure" file.
We can do that in another CL after merging this.

Reply:
Should it be removed?

Reply:
We will just run autoconf after the merge.